ja/tutorials/customize.html

```diff
@@ -0,0 +1,280 @@
+| [language](https://docs.vivliostyle.org/#/ja/create-book#%E4%BD%BF%E7%94%A8%E3%81%99%E3%82%8B%E8%A8%80%E8%AA%9E%E3%81%AE%E6%8C%87%E5%AE%9A) | 使用する言語 |
+| [size](https://docs.vivliostyle.org/#/ja/create-book#%E5%88%A4%E5%9E%8B%E3%81%AE%E6%8C%87%E5%AE%9A) | 用紙サイズ |
+| [theme](https://docs.vivliostyle.org/#/ja/create-book#%E3%83%86%E3%83%BC%E3%83%9E%E3%83%91%E3%83%83%E3%82%B1%E3%83%BC%E3%82%B8%E3%81%AE%E6%8C%87%E5%AE%9A) | テーマパッケージ |
+| entry | 出版物に含める原稿ファイルの一覧 |
+| entryContext | 原稿ファイルがあるディレクトリ |
+| output | 出版物の出力形式（PDF または webpub） |
+| workspaceDir | 中間ファイル（Markdown から作られた HTML）の保存先ディレクトリ |
+| toc | 目次を作成するかどうか |
+| cover | 表紙画像 |
+| vfm | [VFM 記法](https://vivliostyle.github.io/vfm/#/ja/vfm)に関する設定 |
+
+詳細は [Vivliostyle CLI ドキュメント](https://docs.vivliostyle.org/#/ja/vivliostyle-cli#%E6%A7%8B%E6%88%90%E3%83%95%E3%82%A1%E3%82%A4%E3%83%AB-vivliostyleconfigjs)を参照してください。
+
+
+### 原稿ファイル
+
+Markdown 記法を拡張した [VFM 記法](https://vivliostyle.github.io/vfm/#/ja/vfm)で記述できます。HTML による記述も可能です。原稿を変更したい場合は、このファイルを編集したり、新しくファイルを追加したりします。
+
+
+### スタイルファイル
+
+出版物のスタイルを定義するスタイルファイルは SCSS で記述され、CSS にトランスパイルされて使用されます。詳細は [Vivliostyle Themes ドキュメント](https://vivliostyle.github.io/themes/#/ja/spec#scss)を参照してください。
+
+多くの場合、スタイルファイルは複数のファイルからなり、1つのディレクトリに格納されています。例えば、雛形作成時に選択したテーマは themes/packages/@vivliostyle/theme-bunko という名前のディレクトリにダウンロードされています。出版物のスタイルを変更したい場合は、これらのファイルを編集したり、新しくファイルを追加したりします。
+
+
+## カスタマイズ
+
+前回のチュートリアルではサンプル原稿を直接編集して PDF を作成していました。また、スタイルには既存のテーマ @vivliostyle/theme-bunko が適用されていました。今回はサンプル原稿ではなく、自分で新しい原稿を用意してみましょう。さらに、既存テーマをカスタマイズして PDF を作成してみましょう。
+
+
+### 原稿ファイルのカスタマイズ
+
+原稿ファイルは、vivliostyle.config.js の entry で設定した値に対応しています。例えば vivliostyle.config.js で以下のような設定をした場合、
+
+```
+module.exports = {
+  ...
+  entry: [
+    'maegaki.md',
+    'honbun.md',
+    'atogaki.md',
+  ],
+  entryContext: './manuscripts',
+  ...
+}
+```
+
+- 原稿ファイルは maegaki.md と honbun.md と atogaki.md の3つ
+- 原稿ファイルは全て ./manuscripts というディレクトリに含まれている
+
+という条件で PDF が作成されます。
+
+では、この条件で作成される PDF をプレビューしてみましょう。以下のボタンから vivliostyle.config.js および原稿ファイルをダウンロードし、mybook ディレクトリの中に置いてみましょう。
+
+<ol class="list--small">
+  {% include button/download.html url="../../../assets/tutorials/ja/customize/vivliostyle.config.js" text="vivliostyle.config.js" class="button--secondary--medium" %}
+  {% include button/download.html url="../../../assets/tutorials/ja/customize/manuscripts.zip" text="manuscripts.zip" class="button--secondary--medium" %}
+</ol>
+
+```
+mybook/
+├── package.json
+├── vivliostyle.config.js (*既存のファイルを上書き)
+├── manuscripts/ (*)
+│   ├── maegaki.md
+│   ├── honbun.md
+│   └── atogaki.md
+└── themes/
+    └── packages/
+        └── @vivliostyle/
+            └── theme-bunko/
+```
+
+プレビュー機能を起動すると、vivliostyle.config.js で指定した通りの順番で、原稿ファイルが出版物に反映されていることがわかります（反映されない場合は、 `npm run preview` が実行されているターミナルで `Ctrl+C` または `⌘+C` を入力してコマンドの実行を停止し、再度 `npm run preview` を実行してください）。
```

Comment:
「ターミナルで `Ctrl+C` または `⌘+C` を入力してコマンドの実行を停止」とあります。
「`⌘+C`」はMacでのキー操作を書いたものだと思いますが、Macのターミナルで `⌘+C` は「コピー」であって 実行停止にはなりません。

```suggestion
プレビュー機能を起動すると、vivliostyle.config.js で指定した通りの順番で、原稿ファイルが出版物に反映されていることがわかります（反映されない場合は、 `npm run preview` が実行されているターミナルで `Ctrl+C` を入力してコマンドの実行を停止し、再度 `npm run preview` を実行してください）。
```